# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../tinylogin/strip.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- tinylogin-1.4/Makefile.vanilla	2006-11-13 06:10:20.000000000 +0100
+++ tinylogin-1.4/Makefile	2006-11-13 06:10:26.000000000 +0100
@@ -230,7 +230,6 @@
 
 tinylogin: $(PWD_LIB) $(LIBBB_LIB) $(OBJECTS)
 	$(CC) $(LDFLAGS) -o $@ $(OBJECTS) $(PWD_LIB) $(LIBBB_LIB) $(LIBRARIES)
-	$(STRIP)
 
 $(PWD_OBJS): %.o: %.c Config.h include/tinylogin.h include/applets.h Makefile
 	- mkdir -p $(PWD_GRP)
